src/zkevm_specs/evm/table.py

```diff
@@ -300,6 +300,19 @@ class CopyDataTypeTag(IntEnum):
     RlcAcc = auto()
 
 
+class MPTProofType(IntEnum):
+    """
+    Tag for MPT lookup.
+    """
+
+    IsStorageMod = auto()
```

Comment:
Since this is an enum (and not a list of boolean flags), I think the `Is` prefix could be removed in all cases.